_docs-sources/foundations/grunty/architecture.mdx

```diff
@@ -0,0 +1,94 @@
+# Architecture
+
+The Grunty GitHub App is a GitHub App introduced to help reduce the burden of integrating Gruntwork products to GitHub resources. The app is designed to be lightweight and flexible, providing a simple way to get started with Gruntwork products.
+
+## Overview
+
+There are three major components to keep in mind when working with the Grunty GitHub App:
+
+1. **GitHub App**: The Grunty GitHub App itself, which is installed in your GitHub organization.
+2. **Grunty API**: The API that interacts with the GitHub API on behalf of the Grunty GitHub App.
```

Comment:
Could we call this the "Gruntwork App" and clarify that it _exposes_ an API that the GitHub app calls? Calling it an "API that interacts with the GitHub app" is confusing since it sounds like it's reaching out to GitHub, when it's actually the other way around.

Reply:
Actually, better yet, maybe we call it the Gruntwork Platform, and clarify that it's hosted by Gruntwork and exposes an API.

Reply:
Oh, maybe it really is a standalone API? Hopefully my confusion will help give some prompts on how to explain it here.

Reply:
We can rename the app in the docs, but that is what it does. It reaches out to GitHub.